markdown.dtx

```diff
@@ -15994,6 +16192,137 @@ following text:
 %
 % \begin{markdown}
 
+#### Tex Math Renderers
+The \mdef{markdownRendererInlineMath} and \mdef{markdownRendererDisplayMath} macros 
+represent inline and display \TeX{} math. 
+Both macros receive a single argument that corresponds to the tex math content.
+These macros will only be produced, when the \Opt{texMathDollars}
+option is enabled.
+
+% \end{markdown}
+%
+% \iffalse
+
+##### Plain \TeX{} Example {.unnumbered}
+
+Using a text editor, create a text document named `document.tex` with the
+following content:
+``` tex
+\input markdown
+\def\markdownOptionTexMathDollars{true}
+\def\markdownRendererInlineMath#1{$\underline{#1}$}
+\def\markdownRendererDisplayMath#1{$$\textstyle#1$$}
+\markdownBegin
+$E=mc^2$
+
+$$\hat{f} \left ( \xi  \right )= \int_{-\infty}^{\infty} f\left ( x  \right ) e^{-i2\pi \xi x} dx$$
+\markdownEnd
+\bye
+```````
+Next, invoke LuaTeX from the terminal:
+``` sh
+luatex document.tex
+``````
+A PDF document named `document.pdf` should be produced and contain the
+following text:
+
+> $E=mc^2$
+> 
+> $$\hat{f} \left ( \xi  \right )= \int_{-\infty}^{\infty} f\left ( x  \right ) e^{-i2\pi \xi x} dx$$
+
+##### \LaTeX{} Example {.unnumbered}
+
+Using a text editor, create a text document named `document.tex` with the
+following content:
+``` tex
+\documentclass{article}
+\usepackage[texMathDollars]{markdown}
+\def\markdownRendererInlineMath#1{\fbox{\begin{math}#1\end{math}}}
+\def\markdownRendererDisplayMath#1{\begin{equation}#1\end{equation}}
```

Comment:
The `equation` environment is defined by the `amsmath` package, which should be loaded in the example document. It is true that the Markdown package currently loads `amsmath`, but this is not guaranteed.